pkgs/tools/misc/discocss/default.nix

```diff
@@ -17,6 +18,15 @@ stdenvNoCC.mkDerivation rec {
     sha256 = "sha256-afmQCOOZ1MwQkbZZNHYfq2+IRv2eOtYBrGVHzDEbUHw=";
   };
 
+  patches = lib.optional discordAlias [
```

Comment:
If discordAlias is false then this is a noop, right? then we shouldn't apply the patch conditionally.

Reply:
Sure, but why?

Reply:
Either don't hurt.

Reply:
To make sure the patch applies with the next update.

Reply:
It wont. And the patch needs to be removed in the next update.

Reply:
Considering the PR will be merged for the next update that is.